pkg/kubelet/eviction/helpers.go

```diff
@@ -609,9 +609,9 @@ func podRequest(pod *v1.Pod, resourceName v1.ResourceName) resource.Quantity {
 	for i := range pod.Spec.InitContainers {
 		switch resourceName {
 		case v1.ResourceMemory:
-			containerValue.Add(*pod.Spec.Containers[i].Resources.Requests.Memory())
+			initValue.Add(*pod.Spec.InitContainers[i].Resources.Requests.Memory())
```

Comment:
is this logic right? since init containers run sequentially, I thought it was supposed to be `max(max(initContainer), sum(container)`

Reply:
cc @derekwaynecarr

Reply:
that is the approach taken at https://github.com/kubernetes/kubernetes/blob/c9c6901303cd83fc0941c2c9964771404bf9cf9f/pkg/quota/evaluator/core/pods.go#L276-L317

Reply:
@liggitt - Indeed, since they're run sequentially it is the right logic

cc @smarterclayton

Reply:
This should look like https://github.com/kubernetes/kubernetes/blob/master/pkg/api/resource/helpers.go#L51.  @liggitt is  correct above